src/app/screens/MakeInvoice/index.tsx

```diff
@@ -94,6 +94,10 @@ function MakeInvoice() {
             <div>
               {amountEditable ? (
                 <div className="mb-4">
+                  {/* 
```

Comment:
I don't think that is the right place to track todos. Move the issue to the current / next sprint if you think this is important.

Reply:
Hm, I kinda like doing this because if a dev is doing something in that file they also see this even if they don't know about the issue. Also better than todos without an issue, of which we have quite some.

Reply:
Sure, happy to talk